.github/workflows/tests.yml

```diff
@@ -64,19 +64,9 @@ jobs:
     - uses: actions/setup-go@v3
       with:
         go-version: 1.16
-    - env:
-        PGUSER: postgres
-        PGPASSWORD: foobar
-        PGDATABASE: osbuildcomposer
-        PGHOST: localhost
-        PGPORT: 5432
       run: |
-        WORKDIR=$(readlink -f pkg/jobqueue/dbjobqueue/schemas)
-        pushd $(mktemp -d)
-        go mod init temp
-        go get -u github.com/jackc/tern
-        go run github.com/jackc/tern migrate -m "$WORKDIR"
-        popd
+        go install github.com/jackc/tern@latest
+        make migrate-dbjobqueue
```

Comment:
There's still an error here. Because a `-` got removed, the run now belongs to the `uses: actions/setup-go` step. `run` and `uses` can't be used in the same step. So we need a hypen in front of `run`; and maybe with a nice `name: migrate jobqueue`.

Reply:
Thank God I don't need to write YAML on everyday basis. I really miss XML from the 00's, it was verbose, but it was readable. XML was Go of programming languages, I am pretty sure.

I improved the whole block, intendation was off, added names to all steps. Looks much better now:

```
.github/workflows/tests.yml
  1:1       warning  missing document start "---"  (document-start)
  5:1       warning  truthy value should be one of [false, true]  (truthy)
  21:9      warning  comment not indented like content  (comments-indentation)
  33:81     error    line too long (140 > 80 characters)  (line-length)
  35:81     error    line too long (91 > 80 characters)  (line-length)
  161:5     error    wrong indentation: expected 6 but found 4  (indentation)
  167:24    warning  too few spaces before comment  (comments)
  169:46    warning  too few spaces before comment  (comments)
  169:81    error    line too long (134 > 80 characters)  (line-length)
  195:81    error    line too long (189 > 80 characters)  (line-length)
  209:5     error    wrong indentation: expected 6 but found 4  (indentation)
```

(these are not mine)

d=)